build/Dockerfile.openapi_generator

```diff
@@ -0,0 +1,9 @@
+FROM openapitools/openapi-generator-cli
+
+RUN apt-get update \
+    && apt-get install -y python3-pip \
+    && pip3 install black
```

Comment:
Given that different versions of black can give us different results, this should probably install a specific version.

Reply:
I was thinking to keep latest version just as we do in barman. This way there will not be huge changes when we update the version after a long time.